packages/lwc-language-server/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "clean": "rm -rf lib && rm -rf src/resources/sfdx/typings/copied",
     "cleanTestData": "node ../../scripts/removeTestData.js",
-    "build": "tsc -p . && node scripts/build.js && node scripts/convert.js",
+    "build": "tsc -p . && node scripts/convert.js && node scripts/build.js",
```

Comment:
This just flip-flops the steps. Without doing this, making any changes to src/lwc-standard.json takes two builds to eventually get to the lib directory.